src/app.module.ts

```diff
@@ -36,6 +42,12 @@ import { EventEmitterModule } from '@nestjs/event-emitter';
         BlockProviderModule,
     ],
     controllers: [AppController],
-    providers: [AppService],
+    providers: [
+        AppService,
+        {
+            provide: APP_INTERCEPTOR,
+            useClass: CacheInterceptor,
+        },
```

Comment:
This would apply the caching to all endpoints. This isn't something that you want.